src/main/java/com/example/userservice/service/UserService.java

```diff
@@ -0,0 +1,98 @@
+package com.example.userservice.service;
+
+import com.example.userservice.dto.PassportCreateRequest;
+import com.example.userservice.dto.UserCreateRequest;
+import com.example.userservice.entities.Passport;
+import com.example.userservice.entities.Users;
+import com.example.userservice.exception.EmailAlreadyExistsException;
+import com.example.userservice.exception.PassportAlreadyExistsException;
+import com.example.userservice.exception.PhoneAlreadyExistsException;
+import com.example.userservice.repository.PassportRepository;
+import com.example.userservice.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
+
+    private final UserRepository userRepository;
+    private final PassportRepository passportRepository;
+
+    @Transactional
+    public UUID createUser(UserCreateRequest request) {
+        validateEmailUniqueness(request.email());
+        validatePhoneUniqueness(request.phone());
+        validatePassportUniqueness(request.passport());
+
+        Users user = createAndSaveUser(request);
+        createAndSavePassport(request, user);
+
+        return user.getExternalId();
```

Comment:
```suggestion
        return userRepository.save(user).getExternalId();
```
Лучше всего так, метод по сути из 1 строчки может состоять